README.md

```diff
@@ -15,6 +15,7 @@ If you want to see Hypertrace in action, you can quickly start Hypertrace via Do
 ```
 git clone https://github.com/hypertrace/hypertrace.git
 cd hypertrace/docker
+docker-compose pull
```

Comment:
a future optimization will be to not copy/paste so much content from the docker dir into the top-level readme. Something like.. "to see more options including a sample app, check our [docker-compose project](docker/README.md)"

Reply:
agree! 👍🏼